docs/user_guide/Upgrade_SIMP/Version_Maps/6.5.0_6.6.0.inc

```diff
@@ -0,0 +1,228 @@
+
+.. code-block:: yaml
+
+   # These are examples of the sssd::domains entry you might see in hiera.
+
+   # LOCAL is the only entry, remove everything
+   sssd::domains:
+     - LOCAL
+
+   # LOCAL is not the only entry, only remove LOCAL
+   sssd::domains:
+     - LOCAL #remove this
+     - MY_DOMAIN
+     - SOMETHING_ELSE
+
+If there are errors in systems before SIMP 6.6.0 has been fully installed, the
+creation of the :code:`LOCAL` domain can be prevented by setting the following
+in :term:`hiera`:
+
+.. code-block:: yaml
+
+   simp::sssd::client::local_domain: false
+
+NTP Changes
+"""""""""""
+
+The :code:`simp_options::ntpd::servers` parameter has been changed to
+:code:`simp_options::ntp::servers`.
+
+Both :program:`ntpd` and :program:`chronyd` use
+:code:`simp_options::ntp::servers` as the default.
+
+In :term:`hiera`, users must:
+
+* Rename :code:`simp_options::ntpd::servers` to :code:`simp_options::ntp::servers`
+* Create an :code:`alias` for backwards compatibility
+
+Old :term:`hiera`:
+
+.. code-block:: yaml
+
+   simp_options::ntpd::servers:
+   - server1.time.server.org
+   - server2.time.server.org
+   - server3.time.server.org
+   - 127.127.1.0
+
+New :term:`hiera`:
+
+.. code-block:: yaml
+
+   simp_options::ntp::servers:
+   - server1.time.server.org
+   - server2.time.server.org
+   - server3.time.server.org
+   - 127.127.1.0
+   # The following is work around needed during upgrade
+   simp_options::ntpd::servers: "%{alias('simp_options::ntp::servers')}"
+
+
+Update Yum Repositories
+"""""""""""""""""""""""
+
+Migrate to the New Format
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**The following replaces Step 1**, `Update the YUM Repositories`, of the
+:ref:`Incremental Upgrades <ug-incremental-upgrades>` instructions.
+
+The SIMP :term:`YUM` repository has been moved from :file:`/var/www/yum/SIMP` to
+:file:`/var/www/yum/SIMP/<os name>/<os version>` to make supporting multiple
+client operating systems easier.
+
+Updates were made to the  :code:`unpack_dvd` script in :package:`simp-utils` to
+extract the RPMs to the new location.
+
+Use the following as `root` to properly migrate to the new format:
+
+.. code-block:: bash
+
+  # Mount the ISO.
+  mkdir /mnt/iso
+  mount -o loop -t iso9660 <location of iso file>
+
+  # Install the simp_utils RPM.
+  cd /mnt/iso/SIMP/noarch
+  yum install -y ./simp-utils*.rpm || yum update -y ./simp-utils*.rpm
+
+  # Unmount the ISO
+  cd $HOME
+  umount /mnt/iso
+
+  # Extract the ISO.
+  unpack_dvd  -v 7.9.2009  <location of iso file>
+
+  # Remove old files and link to the new files
+  cd /var/www/yum/SIMP
+  rm -rf noarch x86_64
+  ln -s ./CentOS/7/noarch ./noarch
+  ln -s ./CentOS/7/x86_64 ./x86_64
+
+Install the new Puppet GPG Key
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Puppet, Inc. has a new :term:`GPG` key that must be used to install the new
+puppet agent.  Install this key on the YUM server by updating
+:package:`simp-gpgkeys`.
+
+.. code-block:: sh
+
```

Comment:
* had to do a `yum clean all; yum makecache` to see the new version
* should have `-y` argument